components/performing_a_backup.mdx

```diff
@@ -0,0 +1,20 @@
+1.  As a fail-safe, please make an image of your robot's hard drive. You should always be able to restore this image
+    if you need to revert back to your previous configuration.
+
+    - The easiest approach may be to either connect a removable (USB or similar) hard drive to the robot computer, or to unplug
+      the robot hard drive and insert it into a computer or workstation.
```

Comment:
change to "_robot's_" for computer and hard drive

Reply:
Done